packages/babel-register/test/cache.js

```diff
@@ -28,13 +28,13 @@ function resetCache() {
   process.env.BABEL_DISABLE_CACHE = oldBabelDisableCacheValue;
 }
 
-describe.skip("@babel/register - caching", () => {
+describe("@babel/register - caching", () => {
   describe("cache", () => {
     let load, get, save;
 
     beforeEach(() => {
       // Since lib/cache is a singleton we need to fully reload it
-      delete require.cache[require.resolve("../lib/cache")];
+      jest.resetModuleRegistry();
```

Comment:
Isn't this run automatically by jest before each test? (https://github.com/facebook/jest/issues/587#issuecomment-153263356)

Reply:
Maybe it changed? Or it is only run once for each test file? At least the tests weren't working without.

Reply:
Should be https://facebook.github.io/jest/docs/en/jest-object.html#jestresetmodules. `resetModuleRegistry` is a deprecated alias and not publicly documented.

https://github.com/facebook/jest/blob/0b076cdfd6020810fc05fb0f26cdca30d688ead1/packages/jest-runtime/src/index.js#L829-L830

It is not run automatically between tests in a single suite, but each suite (file) is run in isolation

Reply:
Ah thanks, I somewhere found it documented, but now I can't find it anymore :) Will fix it.